Formula/swift-format.rb

```diff
@@ -23,14 +23,19 @@ class SwiftFormat < Formula
   # out of the box on Xcode-only systems due to an incorrect sysroot.
   pour_bottle? only_if: :clt_installed
 
-  depends_on xcode: ["13.3", :build]
+  depends_on xcode: ["14.0", :build]
 
   uses_from_macos "swift"
 
   def install
     system "swift", "build", "--disable-sandbox", "-c", "release"
     bin.install ".build/release/swift-format"
     doc.install "Documentation/Configuration.md"
+
+    if OS.mac?
+      deuniversalize_machos ".build/release/lib_InternalSwiftSyntaxParser.dylib"
+      lib.install ".build/release/lib_InternalSwiftSyntaxParser.dylib"
```

Comment:
This might work by passing `"-Wl,-rpath,#{rpath}"` to `swift`.

If that syntax is not recognised, we can install both into `libexec` and symlink `swift-format` into `bin` (or use `write_exec_script`).

Reply:
Oh, it looks like we can use
```ruby
"-Xlinker", "-rpath", "-Xlinker", rpath
```